manufacturing/enclosed_build/README.md

```diff
@@ -17,7 +17,34 @@ Here is the CAD model for viewing: [autodesk360 link](https://a360.co/2PTyRzh)
 
 This page contains a running list of notes and planned updates that will be organized and added to as we get further into beta design and build.
 
-### Venturi Updates
+The purpose of the Beta Build is to build the complete ventilator in an enclosure.
+
+The Components/Sub assemblies of the Beta Build is as follows:
+
+Top Assy - This acts as the highest assembly, it includes all of the parts and sub assemblies to complete the beta build. (includes image of assy, link to CAD, links to stp files, etc) #TODO
+
+Blower Sub Assembly
+
+Brain Sub Assembly
+
+Venturi Sub Assembly
+
+Pinch Valve Sub Assembly
+
+Int & Pneumatic Sub Assembly
+
+Filter Sub Assembly
+
+
+
```

Comment:
Additional linebreaks are redundant and don't come through in Markdown.
Single line-break is ignored.
Double-line-break is new paragraph.
If you need more space, consider organizing it with additional sections/headings.